go.mod

```diff
@@ -60,7 +60,10 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require go.mozilla.org/sops/v3 v3.7.1
```

Comment:
I would like to see this put in with the rest of the requirements.

Reply:
I do not know why tidy decided to keep it standalone.
Will move and see if tidy is still happy

Reply:
This has now been fixed